### PR TITLE
Document AnimationNodeTimeSeek with clarified usage and code example

### PR DESCRIPTION
**Pull request by smix8**
Document AnimationNodeTimeSeek with clarified usage and code example.

Issue was that users often ask "how can I play my animation from a certain position inside the AnimationTree?" but have no idea from the current documentation what the SeekNode can do and how it is used. The majority didn't even know what a seek command in this context is to begin with.

### CODE REVIEW

**Comment by smix8** [comment]
What did I do to offend the clang-god? Is it the empty line between the different codeblocks? How can I properly add a clang-happy empty line then?

File: doc/classes/AnimationNodeTimeSeek.xml [hunk 1]
```diff
@@ -4,7 +4,27 @@
 		A time-seeking animation node to be used with [AnimationTree].
 	</brief_description>
 	<description>
-		This node can be used to cause a seek command to happen to any sub-children of the graph. After setting the time, this value returns to -1.
+		This node can be used to cause a seek command to happen to any sub-children of the animation graph. Use this node type to play an [Animation] from the start or a certain playback position inside the [AnimationNodeBlendTree]. After setting the time and changing the animation playback the seek node automatically goes into sleep mode on the next process frame by setting its [code]seek_position[/code] value to [code]-1.0[/code].
```
Comment (Calinou) [inline doc/classes/AnimationNodeTimeSeek.xml:7]:
```suggestion
		This node can be used to cause a seek command to happen to any sub-children of the animation graph. Use this node type to play an [Animation] from the start or a certain playback position inside the [AnimationNodeBlendTree]. After setting the time and changing the animation playback, the seek node automatically goes into sleep mode on the next process frame by setting its [code]seek_position[/code] value to [code]-1.0[/code].
```

File: doc/classes/AnimationNodeTimeSeek.xml [hunk 2]
```diff
@@ -4,7 +4,27 @@
...
+		animation_tree.set("parameters/Seek/seek_position", 12.0)
+		# alternative
+		animation_tree["parameters/Seek/seek_position"] = 12.0
+		[/gdscript]
+		[csharp]
+		// play child animation from start:
+		animationTree.Set("parameters/Seek/seek_position", 0.0);
+		
+		// play child animation from 12 sec timestamp
+		animationTree.Set("parameters/Seek/seek_position", 12.0);
```
Comment (Calinou) [inline doc/classes/AnimationNodeTimeSeek.xml:25]: To fix CI checks, remove trailing whitespace:

```suggestion
		# Play child animation from the start.
		animation_tree.set("parameters/Seek/seek_position", 0.0)
		# Alternative syntax (same result as above).
		animation_tree["parameters/Seek/seek_position"] = 0.0

		# Play child animation from 12 second timestamp.
		animation_tree.set("parameters/Seek/seek_position", 12.0)
		# Alternative syntax (same result as above).
		animation_tree["parameters/Seek/seek_position"] = 12.0
		[/gdscript]
		[csharp]
		// Play child animation from the start.
		animationTree.Set("parameters/Seek/seek_position", 0.0);

		// Play child animation from 12 second timestamp.
		animationTree.Set("parameters/Seek/seek_position", 12.0);
```

**Review by Calinou — DISMISSED** [review]

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.3.

**Pull request merged**